eta/core/models.py

```diff
@@ -255,16 +255,86 @@ def init_models_dir(new_models_dir):
     manifest.write_to_dir(new_models_dir)
 
 
-def recommend_paths_for_model(name):
-    '''Recommends a base filename and models directory for the given model
-    name, if possible.
+def publish_public_model(
```

Comment:
Same comment from the other PR about this style.  (Of course, take it or leave it as it meets the guidelines...)